server/api/excused_absence/index.js

```diff
@@ -3,11 +3,42 @@ const controller = require('./excused_absence.controller')
 const auth = require('../../auth/auth.service')
 const router = express.Router()
 
+// GET /api/excused_absences
 router.get('/', auth.isAuthenticated(), controller.index)
+
+// GET /api/excused_absences/admin
 router.get('/admin', auth.hasRole('admin'), controller.admin)
-router.get('/:id', auth.isAuthenticated(), controller.show)
+
+// POST /api/excused_absences
 router.post('/', auth.isAuthenticated(), controller.create)
-router.put('/:id', auth.hasRole('admin'), controller.update)
+
+// PUT /api/excused_absences/:id
+router.put('/:id', auth.isAuthenticated(), controller.update)
+
+// Admin-only PUTs
+// PUT /api/excused_absences/:id
+router.put('/:id/approve', auth.hasRole('admin'), controller.update)
+router.put('/:id/deny', auth.hasRole('admin'), controller.update)
+
 router.delete('/:id', auth.isAuthenticated(), controller.destroy)
 
 module.exports = router
+
+
+// User
```

Comment:
This doesn't seem to be the right place for a to-do list long term. Can you remove before merging?

Reply:
@agundy - yep - definitely not the place. @off-by-1-error please remove and push again - thanks!

Reply:
oops, sorry about that. It's gone now